setup.py

```diff
@@ -56,7 +56,8 @@ def readme():
     # dep that commentjson brings in. Delete once this is merged:
     # https://github.com/vaidik/commentjson/pull/33/files
     'lark-parser>=0.7.1,<0.8.0',
-    'SQLAlchemy>=1.3.11',
+    'SQLAlchemy==1.3.11',
+    'mlflow>=1.9.1',
```

Comment:
I don't see where we use this dependency, can we remove?

Reply:
Right, this was a relic from earlier tests where we were calling the mlflow api in caliban. Will remove.